src/Files.Uwp/App.xaml.cs

```diff
@@ -209,17 +210,17 @@ private static async Task InitializeAppComponentsAsync()
                     AppSettings.DetectQuickLook(),
                     TerminalController.InitializeAsync(),
                     JumpList.InitializeAsync(),
-                    ExternalResourcesHelper.LoadOtherThemesAsync(),
+                    //ExternalResourcesHelper.LoadOtherThemesAsync(), //WINUI3
                     ContextFlyoutItemHelper.CachedNewContextMenuEntries
                 );
 
                 userSettingsService.ReportToAppCenter();
             });
 
             // Check for required updates
-            var updateService = Ioc.Default.GetRequiredService<IUpdateService>();
-            await updateService.CheckForUpdates();
-            await updateService.DownloadMandatoryUpdates();
+            //var updateService = Ioc.Default.GetRequiredService<IUpdateService>();
+            //await updateService.CheckForUpdates(); // WINUI3
+            //await updateService.DownloadMandatoryUpdates();
```

Comment:
Can we re-enable this update service?

Reply:
It crashes the app soon after start, to be checked why.

Reply:
I think this is a blocker :thinking:

Reply:
I can have a look at this and see if I can figure out what's going on.

Reply:
I'm able to run the app in debug mode without any issues but I think it's worth checking.

Reply:
Can't repro the crash anymore 👍

Reply:
Checked again and I can't repro the crash anymore 👍